src/renderer/components/uielements/assets/assetInfo/AssetInfo.tsx

```diff
@@ -57,7 +58,9 @@ export const AssetInfo: React.FC<Props> = (props): JSX.Element => {
         oWalletInfo,
         O.filter(({ walletType }) => isLedgerWallet(walletType)),
         O.map((walletInfo) => (
-          <Styled.WalletType key={walletInfo.address}>{intl.formatMessage({ id: 'ledger.title' })}</Styled.WalletType>
+          <WalletTypeLabel style={{ padding: '1px 3px', marginLeft: 5 }} key={walletInfo.address}>
```

Comment:
Try to avoid to use `style`, but define styles in external file: Add create `WalletTypeLabel` in `AssetInfo.styles.ts` (we do it similar for other UI elements)